### PR TITLE
Fix/43: Builder options to be moved to BuildConfiguration

### PR DESCRIPTION
**Pull request by alphaville**
Fixed issue #43.

### CODE REVIEW

**Comment by korken89** [comment]
Hmm some CI is failing, can you have a look?

**Comment by alphaville** [comment]
@korken89 It's not failing - I cancelled it because Travis would run the same tests both for the branch and for the pull request. I configured it to run the tests only once from now on. See the tests [here](https://travis-ci.org/alphaville/optimization-engine/builds/542870181).

**Comment by korken89** [comment]
Hmm, this is not good though. Maybe I should setup bors for this repo to handle all this for us?

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
...
+<!-- ---------------------
+      Unreleased
+     --------------------- -->
+
+## [Unreleased]
+
+### Fixed
+
+### Added
+
```
Comment (korken89) [inline CHANGELOG.md]: I think the new interface should be in the `Added` section

**Comment by alphaville** [comment]
@korken89 I can reactivate it - it's just double the effort for Travis and the waiting time for us. In any case, we can check the results on Travis and we get an email if anything fails.

**Comment by korken89** [comment]
What is quite common is to not allow pushing to master (only additions through PRs) and then disable push builds.
Makes reviewing additions a lot easier too :)

**Comment by alphaville** [comment]
@korken89 I just protected `master` and configured Travis CI to build only PRs.

**Comment by korken89** [comment]
Perfect! Did you have a chance to look at my requested changes? I'll merge it as soon as it is addressed :)

**Comment by alphaville** [comment]
@korken89 Yes, I just update the CHANGELOG. From my side, we can merge this.

**Comment by korken89** [comment]
<3

**Pull request merged**